resources/roca_lib.brs

```diff
@@ -211,7 +219,7 @@ end function
 ' @param description a string describing the test case
 ' @param suite the suite from the parent 'describe', used to track test pass and fail states
 ' @param func the function to execute as part of the test case
-sub __suite_registerCase(mode as string, description as string, suite as object, func as object)
+sub __suite_registerCase(mode as string, description as string, suite as object, func as object, args as dynamic)
```

Comment:
nit, can you document the new parameter?

Reply:
I think @alimnios72 was asking for a `@param` comment here 😉

Reply:
yeah, I was talking about the `@param` documentation